composer.json

```diff
@@ -3,12 +3,23 @@
     "type": "project",
     "require": {
         "vimeo/psalm": "dev-master",
-        "muglug/simple-markdown-blog": "dev-main"
+        "muglug/simple-markdown-blog": "dev-main",
+        "bamarni/composer-bin-plugin": "^1.4"
     },
     "autoload": {
         "psr-4": {
             "Psalm\\Tests\\Provider\\": "vendor/vimeo/psalm/tests/Provider",
             "PsalmDotOrg\\": "src"
         }
+    },
+    "scripts": {
+        "post-install-cmd": [
+            "composer bin all install",
+            "phinx migrate"
```

Comment:
There is a flag like `-y` to avoid a prompt in production before phinx actually runs any migration.

Reply:
It doesn't ask me any questions, and the only two commands that appear to ask anything are `create` and `seed:create`:

![image](https://user-images.githubusercontent.com/57403/124050656-a3b21680-da23-11eb-90a2-a166eb2e8f15.png)